go-controller/pkg/ovn/ovn.go

```diff
@@ -146,30 +188,22 @@ type Controller struct {
 
 	joinSwIPManager *lsm.JoinSwitchIPManager
 
-	// event recorder used to post events to k8s
-	recorder record.EventRecorder
-
-	// libovsdb northbound client interface
-	nbClient libovsdbclient.Client
-
-	// libovsdb southbound client interface
-	sbClient libovsdbclient.Client
-
 	// v4HostSubnetsUsed keeps track of number of v4 subnets currently assigned to nodes
+	// TBD, need to determine if this metrics is per-controller or global
 	v4HostSubnetsUsed float64
 
 	// v6HostSubnetsUsed keeps track of number of v6 subnets currently assigned to nodes
 	v6HostSubnetsUsed float64
 
 	// Objects for pods that need to be retried
 	retryPods *RetryObjs
+	// Objects for nodes that need to be retried
+	retryNodes *RetryObjs
 
 	// Objects for network policies that need to be retried
 	retryNetworkPolicies *RetryObjs
-
```

Comment:
nit: I would preserve this newline.

Reply:
may I ask what is the significance of the newline, why newline for this but not for others (for example, retryEgressIPs)?

Reply:
Oh none. I thought we were having newlines between variables. Not that I care - but I think whatever we do, it should be consistent - at least with the rest of the file.

I'm (personally) OK either way.